settings.gradle.kts

```diff
@@ -25,7 +25,7 @@ dependencyResolutionManagement {
     repositories {
         google()
         mavenCentral()
-        maven { url = uri("https://jitpack.io") }
+        maven { url = uri("https://a8c-libs.s3.amazonaws.com/android") }
```

Comment:
FYI: This configuration below is what you actually need:

```groovy
        maven {
            url = uri("https://a8c-libs.s3.amazonaws.com/android")
            content {
                includeGroup("com.automattic")
                includeGroup("com.automattic.ucrop")
            }
        }
```

Reply:
hmm what does this add? It works fine without it 🤔

Reply:
Good question @AdamGrzybkowski ! 💯 

So, the purpose of `content/includeGroup` is:
1. **Artifact Filtering**: The `content` block defines which artifacts are allowed to be downloaded from this specific repository.
2. **Group Restrictions**: By using `includeGroup`, the configuration limits the repository to only provide artifacts from the specified group IDs.

So, in our case, the repository configuration I shared above will do the following:
1. **Include com.automattic**: Allow artifacts with the group ID "com.automattic" to be downloaded from this repository.
2. **Include com.automattic.ucrop**: Also permit artifacts with the group ID "com.automattic.ucrop" to be retrieved from this repository.

This setup is in general what we have been applying to all our repos, just to optimize dependency resolution by only checking for the specified groups, prevent unintended artifacts from being downloaded and explicitly define which artifact groups are expected from this particular repository.

As such, this configuration ensures that only artifacts from the `com.automattic` and `com.automattic.ucrop` groups will be considered when resolving dependencies from the `https://a8c-libs.s3.amazonaws.com/android` repository. Any other group IDs will be ignored for this specific repository, even if they exist there.

Does that all make sense to you? 🙏

Reply:
Thanks for the explanation, makes total sense!

Reply:
Awesome, thank YOU for the inquiry! 🥇